packages/cmrjs/cmr-utils.js

```diff
@@ -540,6 +540,7 @@ async function updateUMMGMetadata({
     Bucket: cmrFile.bucket,
     Key: getS3KeyOfFile(cmrFile),
     Body: JSON.stringify(metadataObject),
+    ContentType: 'application/json',
```

Comment:
I have this hard-coded for now because we would expect all `UMMG-json` cmr files to be `json`. Open to arguments against this, though.